pkg/diskmaker/controllers/lvset/device_age.go

```diff
@@ -0,0 +1,52 @@
+package lvset
+
+import (
+	"sync"
+	"time"
+)
+
+var (
+	// deviceMinAge is the minimum age for a device to be considered safe to claim
+	// otherwise, it could be a device that some other entity has attached and we have not claimed.
+	deviceMinAge = time.Minute
+)
+
+type ageMap struct {
+	ageMap  map[string]time.Time
+	mux     sync.Mutex
+	getTime *func() time.Time
```

Comment:
Can we rename this function to `currentTime` which is what actually is. Also add a comment here that this is just for test purpose?